lib/active_data/model/scopes.rb

```diff
@@ -37,11 +37,27 @@ def respond_to_missing?(method, _)
           super || self.class._scope_model.respond_to?(method)
         end
 
-        def method_missing(method, *args, &block)
+        def method_missing(method, *args, **kwargs, &block)
```

Comment:
@rewritten I would define multiple versions of `#method_missing`, by checking `RUBY_VERSION` constant.

The `if` statements are adding performance overhead each time this method is invoked.

Example:

```ruby
if RUBY_VERSION ~= /\A2.6/
  def method_missing # 2.6 compatible implementation
  end
elsif RUBY_VERSION ~= /\A2.7/
    def method_missing # 2.7 compatible implementation
    end
elsif RUBY_VERSION ~= /\A3/
    def method_missing # 3 compatible implementation
    end
end
```